module_2/gifts/repository/src/main/java/com/epam/esm/repository/query_builder/EntityConstant.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.esm.repository.query_builder;
+
+public class EntityConstant {
+	public static final String PARAMS = "params";
+	public static final String PAGE = "page";
+	public static final String LIMIT = "limit";
+	public static final String SEARCH = "search";
+	public static final String ORDER = "sort";
+	public static final String TAG = "tag";
+
+	public static final String NAME = "name";
+	public static final String ID = "id";
+	public static final String IS_DELETED = "is_deleted";
+
+	public static final String CERTIFICATE_DESCRIPTION = "description";
+	public static final String CERTIFICATE_PRICE = "price";
+	public static final String CERTIFICATE_DURATION = "duration";
+	public static final String CERTIFICATE_CREATE_DATE = "create_date";
+	public static final String CERTIFICATE_LAST_UPDATE_DATE = "last_update_date";
+
+	public static final String CERTIFICATE_ID = "certificate_id";
+	public static final String TAG_ID = "tag_id";
+	
+	public static final char DESC_SIGN = '-';
+
```

Comment:
good practice to add a private constructor in cases when all methods or fields a static